var/spack/repos/builtin/packages/libgridxc/package.py

```diff
@@ -8,7 +8,7 @@
 from spack.package import *
 
 
-class Libgridxc(MakefilePackage):
+class Libgridxc(AutotoolsPackage):
```

Comment:
Adding `autotools` is :heavy_check_mark:, since upstream says that the legacy way of building the library will be deprecated. I wonder though if _just_ changing the build system, instead of adding multiple build systems, won't break old version of the library. Can you check that?

Reply:
Also, there is no need to have an `edit` section for `AutotoolsPackage`. I suspect though that we might need to switch build system at a given version.